src/js/containers/search/SearchContainer.jsx

```diff
@@ -72,33 +72,87 @@ export const parseRemoteFilters = (data) => {
 const SearchContainer = ({ history }) => {
     const { urlHash } = useParams();
     const dispatch = useDispatch();
-    const { filters, download, appliedFilters } = useSelector((state) => state);
+    const {
+        filters: stagedFilters,
+        download,
+        appliedFilters: {
+            filters: appliedFilters,
+            _empty: areAppliedFiltersEmpty,
+            _complete: areFiltersApplied
```

Comment:
If we really want to change these variable, which I think we do, we should change this in redux. If this is too big a haul we should add a TODO.